babel.config.js

```diff
@@ -1,5 +1,7 @@
 module.exports = api => {
-  if (api) api.cache(false)
+  if (api && !api.env('test')) {
```

Comment:
Jest must set `api.cache(false)` itself and Babel doesn't like you calling it twice

Reply:
It would be really good if we could remove the complexity from this file. I had to look up `api.cache(false)` to see what the point of it was!

I made another comment about whether we need the tests to be compiled with babel at all – if we didn't, then we wouldn't need any logic in this file, and we could just turn it into a static `module.exports = { ... }` declaration, removing the `api.cache()` thing entirely.